cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -103,7 +103,8 @@ set(configure_command
     ${configure_command})
 
 # Pass C flags through
-set(configure_opts ${configure_opts} ${CMAKE_C_FLAGS})
+string(REGEX REPLACE "[ ]*-f?(PIC|PIE|pic|pie)" "" C_FLAGS_NO_PIC "${CMAKE_C_FLAGS}")
```

Comment:
This regex converts `--fpiesomething` to `-something`.

Reply:
Let me try something better. Sadly, even passing " " to OpenSSL breaks the `Configure` script.